src/index.js

```diff
@@ -12,8 +12,7 @@ export function h(name, props) {
       for (i = node.length; i--; ) {
         stack.push(node[i])
       }
-    } else if (null == node || true === node || false === node) {
```

Comment:
@okwolf Nice try, but this won't work when the text node is `0`.

Reply:
@JorgeBucaran we're missing a test that should fail with this version then.